client/src/websocket/actions.ts

```diff
@@ -109,6 +119,10 @@ export const openWebSocket = () => dispatch => {
   };
 }
 
+const resetPixelColor = () => {
```

Comment:
What is this for? Might need to be removed.

Reply:
yep good catch :)